streamlit_folium/frontend/src/index.tsx

```diff
@@ -91,8 +91,15 @@ function onDraw(e: any) {
 
   if (type === "circle") {
     var center: [number, number] = [layer._latlng.lng, layer._latlng.lat]
-    var radius = layer.options.radius // In km
+    var radius = layer.options.radius // In meters
```

Comment:
Just to be clear, this is just correcting the incorrect comment, right? Not changing the logic?

Reply:
correct, the line is the same, just noticed the comment was incorrect so changed it too.